rfc6962-bis.xml

```diff
@@ -1103,7 +1103,7 @@ messages.
         <t>
           Note that this message is not signed -- the <spanx style="verb">entries</spanx> data can be
 verified by constructing the Merkle Tree Hash corresponding to a retrieved
-STH. All leaves MUST be v1 or v2. However, a compliant v1 client MUST NOT construe an
+STH. All leaves MUST be v1 or v2. However, a compliant v2 client MUST NOT construe an
```

Comment:
There are further problems with this sentence because LogEntryType doesn't exist anymore. Looking into re-writing it.